acquia_lift_profiles/qunit/tests.js

```diff
@@ -177,6 +177,28 @@ QUnit.asyncTest("Get context values no cache", function( assert ) {
 
 });
 
+QUnit.asyncTest("Trigger retrievedSegments", function( assert ) {
+  expect(5);
+  var contextResult = Drupal.personalize.visitor_context.acquia_lift_profiles_context.getContext({'segment1':'segment1', 'segment2':'segment2'});
+  assert.ok(contextResult instanceof Promise);
+  var cached = Drupal.personalize.visitor_context_read('segment1', 'acquia_lift_profiles_context');
```

Comment:
Modified the test to only make sure acquiaLiftStoredSegments is being triggered.